lib/delayed/backend/iron_mq_backend.rb

```diff
@@ -1,18 +1,18 @@
 require_relative 'iron_mq_config'
 
 module Delayed
-  class Worker
+  class IronMqBackend
     class << self
       attr_accessor :config, :ironmq, :queue_name, :delay, :timeout, :expires_in,
```

Comment:
I mean i see number of various uses, e.g. `attr_accessor : queue_name ` means something different than 

```
def queue_name(queue, priority)
   "#{queue}_#{priority || 0}"
end
```

which one is true and how it supposed to work?